tools/kvrocks2redis/parser.cc

```diff
@@ -89,7 +89,7 @@ Status Parser::parseComplexKV(const Slice &ns_key, const Metadata &metadata) {
   read_options.iterate_upper_bound = &upper_bound;
   read_options.fill_cache = false;
 
-  auto iter = db_->NewIterator(read_options);
+  auto iter = std::unique_ptr<rocksdb::Iterator>(db_->NewIterator(read_options));
```

Comment:
```suggestion
  auto iter = DBUtil::UniqueIterator(db_, read_options);
```
and include "db_util.h" : )

Reply:
Thanks! ❤️